funnel/templates/macros.html.jinja2

```diff
@@ -828,7 +828,7 @@
             </div>
           </div>
         </div>
-        <p class="mui--text-headline mui--text-bold">{{ post.title }}</p>
+        <p class="mui--text-headline mui--text-bold"><a class="mui--text-dark" :href="post.urls.project_view">{{ post.title }}</a></p>
```

Comment:
Also needs a link from the date.

Reply:
I have added link to update and date https://github.com/hasgeek/funnel/pull/783/files#diff-a5041c530595e2f582d11e4983d836b2R818